tools/migrate_branch.py

```diff
@@ -252,18 +252,37 @@ def _update_metafiles(self, repo, root_contents):
         target_string = self.gh_target_branch
         source_string_dash = self.gh_source_branch.replace('.', '-')
         target_string_dash = self.gh_target_branch.replace('.', '-')
-        if root_contents.get('README.md'):
-            tree_data.append(self._replace_content(
-                repo, 'README.md',
-                [(source_string, target_string),
-                 (source_string_dash, target_string_dash),
-                 ("\[//]: # \(addons\).*\[//]: # \(end addons\)",
-                  "[//]: # (addons)\n[//]: # (end addons)")]))
-        if root_contents.get('.travis.yml'):
-            tree_data.append(self._replace_content(
-                repo, '.travis.yml',
-                [(source_string, target_string),
-                 (source_string_dash, target_string_dash)]))
+        REPLACES = {
+            'README.md': {
+                None: [
+                    (source_string, target_string),
+                    (source_string_dash, target_string_dash),
+                    ("\[//]: # \(addons\).*\[//]: # \(end addons\)",
+                     "[//]: # (addons)\n[//]: # (end addons)"),
+                ],
+            },
+            '.travis.yml': {
+                None: [
+                    (source_string, target_string),
+                    (source_string_dash, target_string_dash),
+                    (r"(?i)([^\n]+ODOO_REPO=['\"]ODOO[^\n]+)\n([^\n]+"
+                     r"ODOO_REPO=['\"]oca\/ocb[^\n]+)", r'\2\n\1'),
```

Comment:
BTW, this expression that we made, it may not give you the right results if there are more runs with INCLUDE/EXCLUDE right below each other. What was the reason for putting OCB first again?

Reply:
It was for having runbot using OCB 11.0 branch for deployments instead of the missing odoo one. It doesn't make sense right now.